src/Illuminate/Auth/SessionGuard.php

```diff
@@ -461,7 +461,7 @@ protected function ensureRememberTokenIsSet(AuthenticatableContract $user)
     protected function queueRecallerCookie(AuthenticatableContract $user)
     {
         $this->getCookieJar()->queue($this->createRecaller(
-            $user->getAuthIdentifier().'|'.$user->getRememberToken()
+            $user->getAuthIdentifier().'|'.$user->getRememberToken().'|'.$user->getAuthPassword()
```

Comment:
Really sending password here? :/

Reply:
I'm also having trouble understanding how this can be safe, please advise.

Reply:
It's not the password, it's just the hash.

Reply:
Thanks. In that case, this was what confused me in _Authenticatable.php_:

```
public function getAuthPassword()
{
    return $this->password;
}
```